src/EA.Weee.Web/Views/Shared/_EvidenceTransferProgressiveDisclosurePartial.cshtml

```diff
@@ -31,13 +31,20 @@
                     <p class="govuk-body" id="HistoryStatus-@i">@Model.EvidenceNoteHistoryData[i].Status</p>
                 </td>
                 <td class="govuk-table__cell">
-                    @if (Model.IsInternalUser)
+                    @if (Model.IsPrintable)
                     {
-                        @(this.WeeeGds().ActionLinkToNewTab("View", @Url.Action("ViewEvidenceNoteTransfer", "ManageEvidenceNotes", new { area = "admin", evidenceNoteId = Model.EvidenceNoteHistoryData[i].Id, openedInNewTab = true }), string.Format("View evidence note with reference ID {0}", @Model.EvidenceNoteHistoryData[i].ReferenceDisplay)))
+                        <p>&nbsp;</p>
```

Comment:
@GuidoWorkDev The TH or TD shouldnt be output at all if its a PDF.

So you will need to @if (!Model.IsPrintable) around the following tags...

![image](https://user-images.githubusercontent.com/43237115/195286962-5f771dcd-b488-44b0-b73c-46f16b953b0a.png)


![image](https://user-images.githubusercontent.com/43237115/195287007-e41f48e2-9efb-48c1-8fda-424c3ba89a20.png)